realm/realm-library/src/main/java/io/realm/internal/Table.java

```diff
@@ -593,48 +569,6 @@ private boolean isPrimaryKeyColumn(long columnIndex) {
     }
 
     /**
-     * Returns a view sorted by the specified column and order.
-     *
-     * @param columnIndex the column index.
-     * @param sortOrder the sort order.
-     * @return a sorted view.
-     */
-    public TableView getSortedView(long columnIndex, Sort sortOrder){
```

Comment:
Since no one is using it and i don't see any reason it should be used in a near future. If you know any good reason it will be used in this year, please let me know.